### PR TITLE
RIA-3793 Manage fee update for the type appeal not valid

### PR DESCRIPTION
**Pull request by tarun-palisetty**
### JIRA link (if applicable) ###
[RIA-3793](https://tools.hmcts.net/jira/browse/RIA-3793)

### Change description ###
Case-officer and Admin-officer manageFeeUpdate for the type Appeal not valid

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by mkayad — APPROVED** [review]
LGTM

**Review by makowalski — DISMISSED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ManageFeeUpdateMidEvent.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                .getCaseData();
+
+        FeeUpdateReason feeUpdateReason = asylumCase.read(FEE_UPDATE_REASON, FeeUpdateReason.class)
+            .orElseThrow(() -> new IllegalStateException("Fee update reason is not present"));
+        String newFeeAmount = asylumCase.read(NEW_FEE_AMOUNT, String.class)
+            .orElseThrow(() -> new IllegalStateException("New fee amount is not present"));
+
+        if (feeUpdateReason == FeeUpdateReason.APPEAL_NOT_VALID && new BigDecimal(newFeeAmount).signum() > 0) {
+
+            PreSubmitCallbackResponse<AsylumCase> callbackResponse = new PreSubmitCallbackResponse<>(asylumCase);
```
Comment (divisathyan) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ManageFeeUpdateMidEvent.java:59]: It need not be repeated again. Can create one callbackResponse.

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ManageFeeUpdateMidEventTest.java [hunk 2]
```diff
@@ -0,0 +1,136 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.presubmit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.Assert.assertFalse;
```
Comment (divisathyan) [inline src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ManageFeeUpdateMidEventTest.java:5]: Junit4 test used. Please use Junit5 assertions

**Review by divisathyan — COMMENTED** [review]
Please look into comments

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ManageFeeUpdateMidEvent.java (hunk 1 above)
Comment (tarun-palisetty) [reply src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ManageFeeUpdateMidEvent.java]: resolved

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ManageFeeUpdateMidEventTest.java (hunk 2 above)
Comment (tarun-palisetty) [reply src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ManageFeeUpdateMidEventTest.java]: fixed

**Review by divisathyan — DISMISSED** [review]
LGTM

**Review by bradsorour — APPROVED** [review]
LGTM

**Pull request merged**